### PR TITLE
Upgrade to angular 15 

### LINKED ISSUE #2538: [Housekeeping] Angular 15 upgrade
Would be great to get up to date with our Angular version. I believe only very few changes will be necessary, and we will get some nice benefits from the upgrade.
 
We need Angular 14 to start experimenting with standalone components.

### PR DESCRIPTION
**Pull request by kodeaben**
## Which issue does this PR close?

This PR closes #2538 
## Does this PR introduce a breaking change?

- [x] Yes
- [ ] No

## Checklist:

The following tasks should be carried out in sequence in order to follow [the process of contributing](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#the-process-of-contributing) correctly.

### Reminders
- [-] Make sure you have implemented tests following the guidelines in: "[The good: Test](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Test)".
- [-] Make sure you have updated the cookbook with examples and showcases (for bug fixes, enhancements & new components).

### Review  
- [x] Determine if your changes are a fix, feature or breaking-change, and add the matching label to your PR. If it is tooling, dependency updates or similar, add ignore-for-release.
- [x] Do a [self-review](https://github.com/kirbydesign/designsystem/wiki/The-Good%3A-Self-review).
- [x] Request that the changes are code-reviewed 
- [-] Request that the changes are [UX reviewed](https://github.com/kirbydesign/designsystem/blob/main/.github/CONTRIBUTING.md/#ux-review) (only necessary if your PR introduces visual changes)

When the pull request has been approved it will be merged to `develop` by Team Kirby.

### CODE REVIEW

File: .browserslistrc [hunk 1]
```diff
@@ -1,17 +0,0 @@
-# This file is used by the build system to adjust CSS and JS output to support the specified browsers below.
```
Comment (RasmusKjeldgaard) [inline .browserslistrc:1]: Can we investigate why this is removed in Angular 15? Migration stuff?

Comment (RasmusKjeldgaard) [reply .browserslistrc:1]: Would be great to add explanation as comment here or in the PR description

Comment (kodeaben) [reply .browserslistrc:1]: It was removed in Angular 15: 
 https://github.com/angular/angular-cli/releases/tag/15.0.0
 https://github.com/angular/angular-cli/commit/9beb878e2eecd32e499c8af557f22f46548248fc
 
 If we have custom need it should be re-added

Comment (kodeaben) [reply .browserslistrc:1]: We had the default values, so it is valid to remove these :https://angular.io/guide/browser-support

File: MIGRATION.md [hunk 2]
```diff
@@ -1,123 +0,0 @@
-# Migrations
```
Comment (RasmusKjeldgaard) [inline MIGRATION.md]: This file should not be removed, it documents how consumers should migrate major versions.

File: .browserslistrc (hunk 1 above)
Comment (RasmusKjeldgaard) [reply .browserslistrc:1]: Cool, lets just leave this convo unresolved for future use

**Review by RasmusKjeldgaard — CHANGES_REQUESTED** [review]
Just a correction about migration.md, we misunderstood each other. 

The issue to use in the PR description is in the title, it is #2538 😄

**Review by RasmusKjeldgaard — APPROVED** [review]
Good work! Remember to change the title so it looks good on a changelog.

**Pull request merged**